### PR TITLE
update Hapi docset to v16.6 and backfill some older versions

### PR DESCRIPTION
**Pull request by sundeer**

### CODE REVIEW

File: docsets/Hapi/docset.json [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
-    "aliases": ["hapi",
-                "hapijs",
-                "hapi.js"]
+    {
+      "version": "15.2",
+      "archive": "versions/15.2/hapi.tgz"
+    },
+    {
+      "version": "14.2",
+      "archive": "versions/14.2hapi.tgz"
```
Comment (Kapeli) [inline docsets/Hapi/docset.json]: You're missing a `/` here. Should be `versions/14.2/hapi.tgz`.

File: docsets/Hapi/docset.json [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+    },
+    {
+      "version": "13.6",
+      "archive": "versions/13.6/hapi.tgz"
```
Comment (Kapeli) [inline docsets/Hapi/docset.json]: You didn't add this archive. I think you added `versions/13.4/hapi.tgz` instead.

**Comment by sundeer** [comment]
both are typos and have been fixed

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**